2019-10-22 10:47:22.461 [error] <0.285.0> CRASH REPORT Process <0.285.0> with 0 neighbours crashed with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.Enumerable',value => nil} in 'Elixir.Enumerable':'impl_for!'/1 line 1
2019-10-22 10:49:22.798 [error] <0.285.0> CRASH REPORT Process <0.285.0> with 0 neighbours crashed with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.Enumerable',value => nil} in 'Elixir.Enumerable':'impl_for!'/1 line 1
2019-10-22 10:54:35.059 [error] <0.294.0> CRASH REPORT Process <0.294.0> with 0 neighbours crashed with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.Enumerable',value => nil} in 'Elixir.Enumerable':'impl_for!'/1 line 1
2019-10-22 12:17:57.539 [error] <0.285.0> CRASH REPORT Process <0.285.0> with 0 neighbours crashed with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.Enumerable',value => nil} in 'Elixir.Enumerable':'impl_for!'/1 line 1
2019-10-22 12:18:10.090 [error] <0.285.0> CRASH REPORT Process <0.285.0> with 0 neighbours crashed with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.Enumerable',value => nil} in 'Elixir.Enumerable':'impl_for!'/1 line 1
2019-10-22 12:20:08.372 [error] <0.285.0> CRASH REPORT Process <0.285.0> with 0 neighbours crashed with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.Enumerable',value => nil} in 'Elixir.Enumerable':'impl_for!'/1 line 1
2019-10-22 12:23:06.091 [error] <0.285.0> CRASH REPORT Process <0.285.0> with 0 neighbours crashed with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.Enumerable',value => nil} in 'Elixir.Enumerable':'impl_for!'/1 line 1
2019-10-22 12:39:07.805 [error] <0.9324.0>@observer_wx:init:106 CRASH REPORT Process <0.9324.0> with 0 neighbours exited with reason: bad argument in call to erlang:register(observer, <0.9324.0>) in observer_wx:init/1 line 106
2019-10-22 12:39:13.843 [error] <0.9830.0>@observer_wx:init:106 CRASH REPORT Process <0.9830.0> with 0 neighbours exited with reason: bad argument in call to erlang:register(observer, <0.9830.0>) in observer_wx:init/1 line 106
2019-10-22 13:33:01.448 [error] <0.4564.0>@observer_wx:init:106 CRASH REPORT Process <0.4564.0> with 0 neighbours exited with reason: bad argument in call to erlang:register(observer, <0.4564.0>) in observer_wx:init/1 line 106
2019-10-22 13:41:22.912 [error] <0.314.0>@Elixir.GenServer:whereis:1181 gen_server <0.314.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:41:22.912 [error] <0.316.0>@Elixir.GenServer:whereis:1181 gen_server <0.316.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:41:22.912 [error] <0.317.0>@Elixir.GenServer:whereis:1181 gen_server <0.317.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:41:22.912 [error] <0.310.0>@Elixir.GenServer:whereis:1181 gen_server <0.310.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:41:22.912 [error] <0.319.0>@Elixir.GenServer:whereis:1181 gen_server <0.319.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:41:22.912 [error] <0.313.0>@Elixir.GenServer:whereis:1181 gen_server <0.313.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:41:22.912 [error] <0.318.0>@Elixir.GenServer:whereis:1181 gen_server <0.318.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:41:22.913 [error] <0.315.0>@Elixir.GenServer:whereis:1181 gen_server <0.315.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:41:22.913 [error] <0.448.0>@Elixir.GenServer:whereis:1181 gen_server <0.448.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:41:22.913 [error] <0.311.0>@Elixir.GenServer:whereis:1181 gen_server <0.311.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:41:22.913 [error] <0.312.0>@Elixir.GenServer:whereis:1181 gen_server <0.312.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:41:22.922 [error] <0.317.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.317.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:41:22.922 [error] <0.311.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.311.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:41:22.922 [error] <0.310.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.310.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:41:22.922 [error] <0.318.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.318.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:41:22.922 [error] <0.314.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.314.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:41:22.922 [error] <0.316.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.316.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:41:22.922 [error] <0.319.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.319.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:41:22.922 [error] <0.312.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.312.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:41:22.922 [error] <0.448.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.448.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:41:22.923 [error] <0.313.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.313.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:41:22.923 [error] <0.315.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.315.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:41:24.923 [error] <0.453.0>@Elixir.GenServer:whereis:1181 gen_server <0.453.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:41:24.923 [error] <0.452.0>@Elixir.GenServer:whereis:1181 gen_server <0.452.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:41:24.924 [error] <0.454.0>@Elixir.GenServer:whereis:1181 gen_server <0.454.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:41:24.924 [error] <0.450.0>@Elixir.GenServer:whereis:1181 gen_server <0.450.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:41:24.925 [error] <0.459.0>@Elixir.GenServer:whereis:1181 gen_server <0.459.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:41:24.925 [error] <0.451.0>@Elixir.GenServer:whereis:1181 gen_server <0.451.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:41:24.925 [error] <0.454.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.454.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:41:24.925 [error] <0.453.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.453.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:41:24.925 [error] <0.455.0>@Elixir.GenServer:whereis:1181 gen_server <0.455.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:41:24.926 [error] <0.456.0>@Elixir.GenServer:whereis:1181 gen_server <0.456.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:41:24.926 [error] <0.450.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.450.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:41:24.926 [error] <0.458.0>@Elixir.GenServer:whereis:1181 gen_server <0.458.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:41:24.926 [error] <0.457.0>@Elixir.GenServer:whereis:1181 gen_server <0.457.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:41:24.926 [error] <0.452.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.452.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:41:24.927 [error] <0.460.0>@Elixir.GenServer:whereis:1181 gen_server <0.460.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:41:24.927 [error] <0.459.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.459.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:41:24.927 [error] <0.451.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.451.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:41:24.927 [error] <0.455.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.455.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:41:24.927 [error] <0.458.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.458.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:41:24.927 [error] <0.456.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.456.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:41:24.928 [error] <0.457.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.457.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:41:24.929 [error] <0.460.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.460.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:41:26.928 [error] <0.461.0>@Elixir.GenServer:whereis:1181 gen_server <0.461.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:41:26.929 [error] <0.462.0>@Elixir.GenServer:whereis:1181 gen_server <0.462.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:41:26.929 [error] <0.465.0>@Elixir.GenServer:whereis:1181 gen_server <0.465.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:41:26.930 [error] <0.464.0>@Elixir.GenServer:whereis:1181 gen_server <0.464.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:41:26.930 [error] <0.463.0>@Elixir.GenServer:whereis:1181 gen_server <0.463.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:41:26.930 [error] <0.461.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.461.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:41:26.930 [error] <0.462.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.462.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:41:26.931 [error] <0.466.0>@Elixir.GenServer:whereis:1181 gen_server <0.466.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:41:26.931 [error] <0.467.0>@Elixir.GenServer:whereis:1181 gen_server <0.467.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:41:26.932 [error] <0.465.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.465.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:41:26.932 [error] <0.464.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.464.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:41:26.932 [error] <0.468.0>@Elixir.GenServer:whereis:1181 gen_server <0.468.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:41:26.933 [error] <0.463.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.463.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:41:26.933 [error] <0.466.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.466.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:41:26.933 [error] <0.467.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.467.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:41:26.933 [error] <0.468.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.468.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:42:02.512 [error] <0.305.0>@Elixir.GenServer:whereis:1181 gen_server <0.305.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:42:02.521 [error] <0.305.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.305.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:42:04.522 [error] <0.408.0>@Elixir.GenServer:whereis:1181 gen_server <0.408.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:42:04.524 [error] <0.408.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.408.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:42:06.525 [error] <0.409.0>@Elixir.GenServer:whereis:1181 gen_server <0.409.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:42:06.527 [error] <0.409.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.409.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:44:32.897 [error] <0.410.0>@Elixir.GenServer:whereis:1181 gen_server <0.410.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:44:32.899 [error] <0.410.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.410.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:44:34.900 [error] <0.450.0>@Elixir.GenServer:whereis:1181 gen_server <0.450.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:44:34.902 [error] <0.450.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.450.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:44:36.904 [error] <0.451.0>@Elixir.GenServer:whereis:1181 gen_server <0.451.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 13:44:36.906 [error] <0.451.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.451.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 14:23:03.048 [error] <0.296.0>@Elixir.GenServer:whereis:1181 gen_server <0.296.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 14:23:03.059 [error] <0.296.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.296.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 14:23:05.060 [error] <0.486.0>@Elixir.GenServer:whereis:1181 gen_server <0.486.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 14:23:05.062 [error] <0.486.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.486.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 14:31:56.400 [error] <0.305.0>@Elixir.GenServer:whereis:1181 gen_server <0.305.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 14:31:56.401 [error] <0.306.0>@Elixir.GenServer:whereis:1181 gen_server <0.306.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 14:31:56.408 [error] <0.306.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.306.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-22 14:31:56.408 [error] <0.305.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.305.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis([]) line 1181
2019-10-24 14:08:37.180 [error] <0.303.0>@Elixir.KafkaEx.Protocol.ApiVersions:parse_response:62 CRASH REPORT Process <0.303.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.KafkaEx.Protocol.ApiVersions':parse_response(nil, 0) line 62
2019-10-24 15:11:52.607 [error] <0.331.0>@Elixir.GenServer:whereis:1181 gen_server <0.331.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"ospbr">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:11:52.607 [error] <0.332.0>@Elixir.GenServer:whereis:1181 gen_server <0.332.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"eamrb">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:11:52.614 [error] <0.331.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.331.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"ospbr">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:11:52.614 [error] <0.332.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.332.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"eamrb">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:11:54.616 [error] <0.531.0>@Elixir.GenServer:whereis:1181 gen_server <0.531.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"mtjrn">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:11:54.616 [error] <0.532.0>@Elixir.GenServer:whereis:1181 gen_server <0.532.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"cbguj">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:11:54.619 [error] <0.531.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.531.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"mtjrn">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:11:54.619 [error] <0.532.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.532.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"cbguj">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:11:56.621 [error] <0.533.0>@Elixir.GenServer:whereis:1181 gen_server <0.533.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"eagcn">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:11:56.621 [error] <0.534.0>@Elixir.GenServer:whereis:1181 gen_server <0.534.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"fdpez">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:11:56.623 [error] <0.534.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.534.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"fdpez">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:11:56.623 [error] <0.533.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.533.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"eagcn">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:11:58.625 [error] <0.536.0>@Elixir.GenServer:whereis:1181 gen_server <0.536.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"rkimf">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:11:58.626 [error] <0.535.0>@Elixir.GenServer:whereis:1181 gen_server <0.535.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"buahn">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:11:58.627 [error] <0.536.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.536.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"rkimf">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:11:58.628 [error] <0.535.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.535.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"buahn">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:12:00.629 [error] <0.537.0>@Elixir.GenServer:whereis:1181 gen_server <0.537.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"bprnv">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:12:00.631 [error] <0.538.0>@Elixir.GenServer:whereis:1181 gen_server <0.538.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"zdlpc">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:12:00.632 [error] <0.537.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.537.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"bprnv">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:12:00.633 [error] <0.538.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.538.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"zdlpc">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:12:02.633 [error] <0.539.0>@Elixir.GenServer:whereis:1181 gen_server <0.539.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"poufm">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:12:02.635 [error] <0.540.0>@Elixir.GenServer:whereis:1181 gen_server <0.540.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"cpmul">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:12:02.636 [error] <0.539.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.539.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"poufm">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:12:02.637 [error] <0.540.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.540.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"cpmul">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:12:04.638 [error] <0.541.0>@Elixir.GenServer:whereis:1181 gen_server <0.541.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"pfvbu">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:12:04.639 [error] <0.542.0>@Elixir.GenServer:whereis:1181 gen_server <0.542.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"tbphl">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:12:04.640 [error] <0.541.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.541.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"pfvbu">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:12:04.642 [error] <0.542.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.542.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"tbphl">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:12:06.642 [error] <0.543.0>@Elixir.GenServer:whereis:1181 gen_server <0.543.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"fprlv">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:12:06.645 [error] <0.543.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.543.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"fprlv">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:12:06.645 [error] <0.544.0>@Elixir.GenServer:whereis:1181 gen_server <0.544.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"sgtmn">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:12:06.647 [error] <0.544.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.544.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"sgtmn">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:12:08.646 [error] <0.545.0>@Elixir.GenServer:whereis:1181 gen_server <0.545.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"lmczr">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:12:08.648 [error] <0.545.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.545.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"lmczr">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:12:08.649 [error] <0.546.0>@Elixir.GenServer:whereis:1181 gen_server <0.546.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"hgvfl">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:12:08.650 [error] <0.546.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.546.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"hgvfl">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:12:10.650 [error] <0.547.0>@Elixir.GenServer:whereis:1181 gen_server <0.547.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"amgdk">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:12:10.652 [error] <0.547.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.547.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"amgdk">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:12:10.653 [error] <0.548.0>@Elixir.GenServer:whereis:1181 gen_server <0.548.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"mapcf">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:12:10.655 [error] <0.548.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.548.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"mapcf">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:12:12.654 [error] <0.549.0>@Elixir.GenServer:whereis:1181 gen_server <0.549.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"febpj">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:12:12.656 [error] <0.549.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.549.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"febpj">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:12:12.657 [error] <0.550.0>@Elixir.GenServer:whereis:1181 gen_server <0.550.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"rvteo">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:12:12.659 [error] <0.550.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.550.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"rvteo">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:12:14.660 [error] <0.551.0>@Elixir.GenServer:whereis:1181 gen_server <0.551.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"edoic">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:12:14.662 [error] <0.551.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.551.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"edoic">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:12:14.663 [error] <0.552.0>@Elixir.GenServer:whereis:1181 gen_server <0.552.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"oktsj">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:12:14.664 [error] <0.552.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.552.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"oktsj">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:12:16.667 [error] <0.553.0>@Elixir.GenServer:whereis:1181 gen_server <0.553.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"ekaup">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:12:16.669 [error] <0.554.0>@Elixir.GenServer:whereis:1181 gen_server <0.554.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"uijgt">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:12:16.669 [error] <0.553.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.553.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"ekaup">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:12:16.670 [error] <0.554.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.554.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"uijgt">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:12:18.672 [error] <0.555.0>@Elixir.GenServer:whereis:1181 gen_server <0.555.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"bngsj">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:12:18.673 [error] <0.556.0>@Elixir.GenServer:whereis:1181 gen_server <0.556.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"zfpme">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:12:18.674 [error] <0.555.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.555.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"bngsj">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:12:18.675 [error] <0.556.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.556.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"zfpme">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:31:31.954 [error] <0.310.0>@Elixir.GenServer:whereis:1181 gen_server <0.310.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"shlpf">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:31:31.955 [error] <0.309.0>@Elixir.GenServer:whereis:1181 gen_server <0.309.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"mzgjr">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:31:31.966 [error] <0.309.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.309.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"mzgjr">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:31:31.966 [error] <0.310.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.310.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"shlpf">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:31:33.967 [error] <0.420.0>@Elixir.GenServer:whereis:1181 gen_server <0.420.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"gnpze">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:31:33.968 [error] <0.419.0>@Elixir.GenServer:whereis:1181 gen_server <0.419.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"zikhn">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:31:33.970 [error] <0.419.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.419.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"zikhn">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:31:33.970 [error] <0.420.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.420.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"gnpze">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:33:08.644 [error] <0.298.0>@Elixir.GenServer:whereis:1181 gen_server <0.298.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"vgsbr">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:33:08.645 [error] <0.297.0>@Elixir.GenServer:whereis:1181 gen_server <0.297.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"evzls">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:33:08.660 [error] <0.298.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.298.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"vgsbr">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:33:08.660 [error] <0.297.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.297.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"evzls">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:49:17.458 [error] <0.310.0>@Elixir.GenServer:whereis:1181 gen_server <0.310.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"lkvbg">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:49:17.458 [error] <0.307.0>@Elixir.GenServer:whereis:1181 gen_server <0.307.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"amgus">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:49:17.459 [error] <0.308.0>@Elixir.GenServer:whereis:1181 gen_server <0.308.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"udnlz">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:49:17.459 [error] <0.309.0>@Elixir.GenServer:whereis:1181 gen_server <0.309.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"hkjtn">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:49:17.459 [error] <0.306.0>@Elixir.GenServer:whereis:1181 gen_server <0.306.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"mlzvp">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:49:17.471 [error] <0.308.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.308.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"udnlz">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:49:17.471 [error] <0.305.0>@Elixir.GenServer:whereis:1181 Supervisor {<0.305.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.308.0> exit with reason no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"udnlz">>,partition => 1,topic => ...}) line 1181 in context child_terminated
2019-10-24 15:49:17.471 [error] <0.307.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.307.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"amgus">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:49:17.472 [error] <0.306.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.306.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"mlzvp">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:49:17.472 [error] <0.305.0>@Elixir.GenServer:whereis:1181 Supervisor {<0.305.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.307.0> exit with reason no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"amgus">>,partition => 1,topic => ...}) line 1181 in context child_terminated
2019-10-24 15:49:17.472 [error] <0.305.0>@Elixir.GenServer:whereis:1181 Supervisor {<0.305.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.306.0> exit with reason no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"mlzvp">>,partition => 1,topic => ...}) line 1181 in context child_terminated
2019-10-24 15:49:17.472 [error] <0.310.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.310.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"lkvbg">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:49:17.472 [error] <0.309.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.309.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"hkjtn">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:49:17.472 [error] <0.305.0>@Elixir.GenServer:whereis:1181 Supervisor {<0.305.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.310.0> exit with reason no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"lkvbg">>,partition => 1,topic => ...}) line 1181 in context child_terminated
2019-10-24 15:49:17.472 [error] <0.305.0>@Elixir.GenServer:whereis:1181 Supervisor {<0.305.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.309.0> exit with reason no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"hkjtn">>,partition => 1,topic => ...}) line 1181 in context child_terminated
2019-10-24 15:49:19.477 [error] <0.424.0>@Elixir.GenServer:whereis:1181 gen_server <0.424.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"uifnr">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:49:19.477 [error] <0.425.0>@Elixir.GenServer:whereis:1181 gen_server <0.425.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"uztpr">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:49:19.478 [error] <0.426.0>@Elixir.GenServer:whereis:1181 gen_server <0.426.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"jofzh">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:49:19.479 [error] <0.426.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.426.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"jofzh">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:49:19.479 [error] <0.305.0>@Elixir.GenServer:whereis:1181 Supervisor {<0.305.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.426.0> exit with reason no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"jofzh">>,partition => 1,topic => ...}) line 1181 in context child_terminated
2019-10-24 15:49:19.479 [error] <0.427.0>@Elixir.GenServer:whereis:1181 gen_server <0.427.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"fmusv">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:49:19.480 [error] <0.424.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.424.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"uifnr">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:49:19.480 [error] <0.425.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.425.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"uztpr">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:49:19.480 [error] <0.428.0>@Elixir.GenServer:whereis:1181 gen_server <0.428.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"facej">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:49:19.481 [error] <0.427.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.427.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"fmusv">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:49:19.481 [error] <0.305.0>@Elixir.GenServer:whereis:1181 Supervisor {<0.305.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.424.0> exit with reason no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"uifnr">>,partition => 1,topic => ...}) line 1181 in context child_terminated
2019-10-24 15:49:19.481 [error] <0.305.0>@Elixir.GenServer:whereis:1181 Supervisor {<0.305.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.425.0> exit with reason no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"uztpr">>,partition => 1,topic => ...}) line 1181 in context child_terminated
2019-10-24 15:49:19.482 [error] <0.305.0>@Elixir.GenServer:whereis:1181 Supervisor {<0.305.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.427.0> exit with reason no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"fmusv">>,partition => 1,topic => ...}) line 1181 in context child_terminated
2019-10-24 15:49:19.482 [error] <0.428.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.428.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"facej">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:49:19.482 [error] <0.305.0>@Elixir.GenServer:whereis:1181 Supervisor {<0.305.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.428.0> exit with reason no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"facej">>,partition => 1,topic => ...}) line 1181 in context child_terminated
2019-10-24 15:49:21.482 [error] <0.430.0>@Elixir.GenServer:whereis:1181 gen_server <0.430.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"hjbei">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:49:21.482 [error] <0.429.0>@Elixir.GenServer:whereis:1181 gen_server <0.429.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"jgipr">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:49:21.484 [error] <0.431.0>@Elixir.GenServer:whereis:1181 gen_server <0.431.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"hsrmz">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:49:21.485 [error] <0.429.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.429.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"jgipr">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:49:21.486 [error] <0.305.0>@Elixir.GenServer:whereis:1181 Supervisor {<0.305.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.429.0> exit with reason no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"jgipr">>,partition => 1,topic => ...}) line 1181 in context child_terminated
2019-10-24 15:49:21.486 [error] <0.430.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.430.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"hjbei">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:49:21.487 [error] <0.305.0>@Elixir.GenServer:whereis:1181 Supervisor {<0.305.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.430.0> exit with reason no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"hjbei">>,partition => 1,topic => ...}) line 1181 in context child_terminated
2019-10-24 15:49:21.487 [error] <0.431.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.431.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"hsrmz">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:49:21.487 [error] <0.305.0>@Elixir.GenServer:whereis:1181 Supervisor {<0.305.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.431.0> exit with reason no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"hsrmz">>,partition => 1,topic => ...}) line 1181 in context child_terminated
2019-10-24 15:49:21.488 [error] <0.432.0>@Elixir.GenServer:whereis:1181 gen_server <0.432.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"lthsv">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:49:21.488 [error] <0.433.0>@Elixir.GenServer:whereis:1181 gen_server <0.433.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"bgivp">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:49:21.488 [error] <0.432.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.432.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"lthsv">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:49:21.489 [error] <0.305.0>@Elixir.GenServer:whereis:1181 Supervisor {<0.305.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.432.0> exit with reason no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"lthsv">>,partition => 1,topic => ...}) line 1181 in context child_terminated
2019-10-24 15:49:21.489 [error] <0.433.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.433.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"bgivp">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:49:21.489 [error] <0.305.0>@Elixir.GenServer:whereis:1181 Supervisor {<0.305.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.433.0> exit with reason no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"bgivp">>,partition => 1,topic => ...}) line 1181 in context child_terminated
2019-10-24 15:52:35.574 [error] <0.301.0> gen_server <0.301.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"vegit">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3
2019-10-24 15:52:35.575 [error] <0.300.0> gen_server <0.300.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"hsdeb">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3
2019-10-24 15:52:35.575 [error] <0.299.0> gen_server <0.299.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"pknrs">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3
2019-10-24 15:52:35.575 [error] <0.298.0> gen_server <0.298.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"mfrhp">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3
2019-10-24 15:52:35.575 [error] <0.297.0> gen_server <0.297.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"sbgrv">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3
2019-10-24 15:52:35.596 [error] <0.298.0> CRASH REPORT Process <0.298.0> with 0 neighbours crashed with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"mfrhp">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3
2019-10-24 15:52:35.596 [error] <0.301.0> CRASH REPORT Process <0.301.0> with 0 neighbours crashed with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"vegit">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3
2019-10-24 15:52:35.596 [error] <0.299.0> CRASH REPORT Process <0.299.0> with 0 neighbours crashed with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"pknrs">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3
2019-10-24 15:52:35.596 [error] <0.296.0> Supervisor {<0.296.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.298.0> exit with reason #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"mfrhp">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3 in context child_terminated
2019-10-24 15:52:35.597 [error] <0.296.0> Supervisor {<0.296.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.301.0> exit with reason #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"vegit">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3 in context child_terminated
2019-10-24 15:52:35.597 [error] <0.300.0> CRASH REPORT Process <0.300.0> with 0 neighbours crashed with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"hsdeb">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3
2019-10-24 15:52:35.597 [error] <0.297.0> CRASH REPORT Process <0.297.0> with 0 neighbours crashed with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"sbgrv">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3
2019-10-24 15:52:35.597 [error] <0.296.0> Supervisor {<0.296.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.299.0> exit with reason #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"pknrs">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3 in context child_terminated
2019-10-24 15:52:35.597 [error] <0.296.0> Supervisor {<0.296.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.300.0> exit with reason #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"hsdeb">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3 in context child_terminated
2019-10-24 15:52:35.597 [error] <0.296.0> Supervisor {<0.296.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.297.0> exit with reason #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"sbgrv">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3 in context child_terminated
2019-10-24 15:52:37.598 [error] <0.10311.0> gen_server <0.10311.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"mtius">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3
2019-10-24 15:52:37.599 [error] <0.10312.0> gen_server <0.10312.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"pcmvj">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3
2019-10-24 15:52:37.600 [error] <0.10314.0> gen_server <0.10314.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"pdzma">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3
2019-10-24 15:52:37.600 [error] <0.10315.0> gen_server <0.10315.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"genul">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3
2019-10-24 15:52:37.600 [error] <0.10313.0> gen_server <0.10313.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"lmjsu">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3
2019-10-24 15:52:37.600 [error] <0.10311.0> CRASH REPORT Process <0.10311.0> with 0 neighbours crashed with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"mtius">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3
2019-10-24 15:52:37.601 [error] <0.296.0> Supervisor {<0.296.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.10311.0> exit with reason #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"mtius">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3 in context child_terminated
2019-10-24 15:52:37.601 [error] <0.10314.0> CRASH REPORT Process <0.10314.0> with 0 neighbours crashed with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"pdzma">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3
2019-10-24 15:52:37.601 [error] <0.10315.0> CRASH REPORT Process <0.10315.0> with 0 neighbours crashed with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"genul">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3
2019-10-24 15:52:37.602 [error] <0.296.0> Supervisor {<0.296.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.10314.0> exit with reason #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"pdzma">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3 in context child_terminated
2019-10-24 15:52:37.602 [error] <0.296.0> Supervisor {<0.296.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.10315.0> exit with reason #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"genul">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3 in context child_terminated
2019-10-24 15:52:37.603 [error] <0.10312.0> CRASH REPORT Process <0.10312.0> with 0 neighbours crashed with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"pcmvj">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3
2019-10-24 15:52:37.603 [error] <0.296.0> Supervisor {<0.296.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.10312.0> exit with reason #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"pcmvj">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3 in context child_terminated
2019-10-24 15:52:37.603 [error] <0.10313.0> CRASH REPORT Process <0.10313.0> with 0 neighbours crashed with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"lmjsu">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3
2019-10-24 15:52:37.604 [error] <0.296.0> Supervisor {<0.296.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.10313.0> exit with reason #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"lmjsu">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3 in context child_terminated
2019-10-24 15:53:20.409 [error] <0.309.0> gen_server <0.309.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"cdgri">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3
2019-10-24 15:53:20.409 [error] <0.313.0> gen_server <0.313.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"ekjlz">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3
2019-10-24 15:53:20.409 [error] <0.312.0> gen_server <0.312.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"nztfc">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3
2019-10-24 15:53:20.409 [error] <0.310.0> gen_server <0.310.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"imfpn">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3
2019-10-24 15:53:20.409 [error] <0.311.0> gen_server <0.311.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"lcfpz">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3
2019-10-24 15:53:20.419 [error] <0.310.0> CRASH REPORT Process <0.310.0> with 0 neighbours crashed with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"imfpn">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3
2019-10-24 15:53:20.419 [error] <0.308.0> Supervisor {<0.308.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.310.0> exit with reason #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"imfpn">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3 in context child_terminated
2019-10-24 15:53:20.419 [error] <0.313.0> CRASH REPORT Process <0.313.0> with 0 neighbours crashed with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"ekjlz">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3
2019-10-24 15:53:20.419 [error] <0.309.0> CRASH REPORT Process <0.309.0> with 0 neighbours crashed with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"cdgri">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3
2019-10-24 15:53:20.419 [error] <0.312.0> CRASH REPORT Process <0.312.0> with 0 neighbours crashed with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"nztfc">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3
2019-10-24 15:53:20.420 [error] <0.308.0> Supervisor {<0.308.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.313.0> exit with reason #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"ekjlz">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3 in context child_terminated
2019-10-24 15:53:20.420 [error] <0.308.0> Supervisor {<0.308.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.309.0> exit with reason #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"cdgri">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3 in context child_terminated
2019-10-24 15:53:20.420 [error] <0.308.0> Supervisor {<0.308.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.312.0> exit with reason #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"nztfc">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3 in context child_terminated
2019-10-24 15:53:20.420 [error] <0.311.0> CRASH REPORT Process <0.311.0> with 0 neighbours crashed with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"lcfpz">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3
2019-10-24 15:53:20.421 [error] <0.308.0> Supervisor {<0.308.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.311.0> exit with reason #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"lcfpz">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3 in context child_terminated
2019-10-24 15:53:22.420 [error] <0.423.0> gen_server <0.423.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"zkmhs">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3
2019-10-24 15:53:22.421 [error] <0.424.0> gen_server <0.424.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"jcgnu">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3
2019-10-24 15:53:22.421 [error] <0.422.0> gen_server <0.422.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"bgcmn">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3
2019-10-24 15:53:22.422 [error] <0.426.0> gen_server <0.426.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"cuakh">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3
2019-10-24 15:53:22.422 [error] <0.423.0> CRASH REPORT Process <0.423.0> with 0 neighbours crashed with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"zkmhs">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3
2019-10-24 15:53:22.423 [error] <0.425.0> gen_server <0.425.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"ghlcp">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3
2019-10-24 15:53:22.423 [error] <0.308.0> Supervisor {<0.308.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.423.0> exit with reason #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"zkmhs">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3 in context child_terminated
2019-10-24 15:53:22.424 [error] <0.422.0> CRASH REPORT Process <0.422.0> with 0 neighbours crashed with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"bgcmn">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3
2019-10-24 15:53:22.424 [error] <0.424.0> CRASH REPORT Process <0.424.0> with 0 neighbours crashed with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"jcgnu">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3
2019-10-24 15:53:22.425 [error] <0.308.0> Supervisor {<0.308.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.422.0> exit with reason #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"bgcmn">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3 in context child_terminated
2019-10-24 15:53:22.425 [error] <0.308.0> Supervisor {<0.308.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.424.0> exit with reason #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"jcgnu">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3 in context child_terminated
2019-10-24 15:53:22.425 [error] <0.426.0> CRASH REPORT Process <0.426.0> with 0 neighbours crashed with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"cuakh">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3
2019-10-24 15:53:22.426 [error] <0.308.0> Supervisor {<0.308.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.426.0> exit with reason #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"cuakh">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3 in context child_terminated
2019-10-24 15:53:22.426 [error] <0.425.0> CRASH REPORT Process <0.425.0> with 0 neighbours crashed with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"ghlcp">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3
2019-10-24 15:53:22.426 [error] <0.308.0> Supervisor {<0.308.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.425.0> exit with reason #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"ghlcp">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3 in context child_terminated
2019-10-24 15:53:51.371 [error] <0.299.0> gen_server <0.299.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"rtlsz">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3
2019-10-24 15:53:51.371 [error] <0.297.0> gen_server <0.297.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"vmplb">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3
2019-10-24 15:53:51.372 [error] <0.300.0> gen_server <0.300.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"mkbct">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3
2019-10-24 15:53:51.372 [error] <0.301.0> gen_server <0.301.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"dkohn">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3
2019-10-24 15:53:51.372 [error] <0.298.0> gen_server <0.298.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"ofutj">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3
2019-10-24 15:53:51.390 [error] <0.298.0> CRASH REPORT Process <0.298.0> with 0 neighbours crashed with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"ofutj">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3
2019-10-24 15:53:51.391 [error] <0.301.0> CRASH REPORT Process <0.301.0> with 0 neighbours crashed with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"dkohn">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3
2019-10-24 15:53:51.391 [error] <0.300.0> CRASH REPORT Process <0.300.0> with 0 neighbours crashed with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"mkbct">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3
2019-10-24 15:53:51.391 [error] <0.296.0> Supervisor {<0.296.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.298.0> exit with reason #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"ofutj">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3 in context child_terminated
2019-10-24 15:53:51.391 [error] <0.297.0> CRASH REPORT Process <0.297.0> with 0 neighbours crashed with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"vmplb">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3
2019-10-24 15:53:51.391 [error] <0.299.0> CRASH REPORT Process <0.299.0> with 0 neighbours crashed with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"rtlsz">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3
2019-10-24 15:53:51.391 [error] <0.296.0> Supervisor {<0.296.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.301.0> exit with reason #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"dkohn">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3 in context child_terminated
2019-10-24 15:53:51.392 [error] <0.296.0> Supervisor {<0.296.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.300.0> exit with reason #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"mkbct">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3 in context child_terminated
2019-10-24 15:53:51.392 [error] <0.296.0> Supervisor {<0.296.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.297.0> exit with reason #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"vmplb">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3 in context child_terminated
2019-10-24 15:53:51.392 [error] <0.296.0> Supervisor {<0.296.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.299.0> exit with reason #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"rtlsz">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3 in context child_terminated
2019-10-24 15:53:53.392 [error] <0.410.0> gen_server <0.410.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"rtszf">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3
2019-10-24 15:53:53.392 [error] <0.411.0> gen_server <0.411.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"uvzso">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3
2019-10-24 15:53:53.393 [error] <0.412.0> gen_server <0.412.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"jdgzn">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3
2019-10-24 15:53:53.394 [error] <0.411.0> CRASH REPORT Process <0.411.0> with 0 neighbours crashed with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"uvzso">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3
2019-10-24 15:53:53.394 [error] <0.410.0> CRASH REPORT Process <0.410.0> with 0 neighbours crashed with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"rtszf">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3
2019-10-24 15:53:53.395 [error] <0.296.0> Supervisor {<0.296.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.411.0> exit with reason #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"uvzso">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3 in context child_terminated
2019-10-24 15:53:53.395 [error] <0.296.0> Supervisor {<0.296.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.410.0> exit with reason #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"rtszf">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3 in context child_terminated
2019-10-24 15:53:53.396 [error] <0.414.0> gen_server <0.414.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"udhea">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3
2019-10-24 15:53:53.396 [error] <0.413.0> gen_server <0.413.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"ejvzc">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3
2019-10-24 15:53:53.396 [error] <0.412.0> CRASH REPORT Process <0.412.0> with 0 neighbours crashed with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"jdgzn">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3
2019-10-24 15:53:53.397 [error] <0.296.0> Supervisor {<0.296.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.412.0> exit with reason #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"jdgzn">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3 in context child_terminated
2019-10-24 15:53:53.397 [error] <0.414.0> CRASH REPORT Process <0.414.0> with 0 neighbours crashed with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"udhea">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3
2019-10-24 15:53:53.397 [error] <0.296.0> Supervisor {<0.296.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.414.0> exit with reason #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"udhea">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3 in context child_terminated
2019-10-24 15:53:53.397 [error] <0.413.0> CRASH REPORT Process <0.413.0> with 0 neighbours crashed with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"ejvzc">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3
2019-10-24 15:53:53.398 [error] <0.296.0> Supervisor {<0.296.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.413.0> exit with reason #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => {name,<<"ejvzc">>}} in 'Elixir.String.Chars':'impl_for!'/1 line 3 in context child_terminated
2019-10-24 15:55:13.091 [error] <0.312.0>@Elixir.GenServer:whereis:1181 gen_server <0.312.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"mfivo">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:55:13.091 [error] <0.313.0>@Elixir.GenServer:whereis:1181 gen_server <0.313.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"eclks">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:55:13.092 [error] <0.311.0>@Elixir.GenServer:whereis:1181 gen_server <0.311.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"cuomf">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:55:13.092 [error] <0.309.0>@Elixir.GenServer:whereis:1181 gen_server <0.309.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"oncah">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:55:13.092 [error] <0.310.0>@Elixir.GenServer:whereis:1181 gen_server <0.310.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"vohsp">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:55:13.105 [error] <0.312.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.312.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"mfivo">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:55:13.105 [error] <0.310.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.310.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"vohsp">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:55:13.105 [error] <0.308.0>@Elixir.GenServer:whereis:1181 Supervisor {<0.308.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.312.0> exit with reason no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"mfivo">>,partition => 1,topic => ...}) line 1181 in context child_terminated
2019-10-24 15:55:13.105 [error] <0.308.0>@Elixir.GenServer:whereis:1181 Supervisor {<0.308.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.310.0> exit with reason no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"vohsp">>,partition => 1,topic => ...}) line 1181 in context child_terminated
2019-10-24 15:55:13.105 [error] <0.313.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.313.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"eclks">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:55:13.105 [error] <0.308.0>@Elixir.GenServer:whereis:1181 Supervisor {<0.308.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.313.0> exit with reason no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"eclks">>,partition => 1,topic => ...}) line 1181 in context child_terminated
2019-10-24 15:55:13.105 [error] <0.309.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.309.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"oncah">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:55:13.105 [error] <0.311.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.311.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"cuomf">>,partition => 1,topic => ...}) line 1181
2019-10-24 15:55:13.105 [error] <0.308.0>@Elixir.GenServer:whereis:1181 Supervisor {<0.308.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.309.0> exit with reason no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"oncah">>,partition => 1,topic => ...}) line 1181 in context child_terminated
2019-10-24 15:55:13.106 [error] <0.308.0>@Elixir.GenServer:whereis:1181 Supervisor {<0.308.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.311.0> exit with reason no function clause matching 'Elixir.GenServer':whereis(#{'__struct__' => 'Elixir.Kafka.Producer.Producer.State',name => <<"cuomf">>,partition => 1,topic => ...}) line 1181 in context child_terminated
2019-10-24 15:56:27.385 [error] <0.309.0>@Elixir.GenServer:whereis:1181 gen_server <0.309.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(<<"ukago">>) line 1181
2019-10-24 15:56:27.386 [error] <0.311.0>@Elixir.GenServer:whereis:1181 gen_server <0.311.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(<<"sinfl">>) line 1181
2019-10-24 15:56:27.386 [error] <0.312.0>@Elixir.GenServer:whereis:1181 gen_server <0.312.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(<<"nackj">>) line 1181
2019-10-24 15:56:27.386 [error] <0.310.0>@Elixir.GenServer:whereis:1181 gen_server <0.310.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(<<"czktd">>) line 1181
2019-10-24 15:56:27.386 [error] <0.313.0>@Elixir.GenServer:whereis:1181 gen_server <0.313.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(<<"fgbcd">>) line 1181
2019-10-24 15:56:27.399 [error] <0.309.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.309.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(<<"ukago">>) line 1181
2019-10-24 15:56:27.399 [error] <0.310.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.310.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(<<"czktd">>) line 1181
2019-10-24 15:56:27.399 [error] <0.312.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.312.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(<<"nackj">>) line 1181
2019-10-24 15:56:27.399 [error] <0.308.0>@Elixir.GenServer:whereis:1181 Supervisor {<0.308.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.309.0> exit with reason no function clause matching 'Elixir.GenServer':whereis(<<"ukago">>) line 1181 in context child_terminated
2019-10-24 15:56:27.399 [error] <0.311.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.311.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(<<"sinfl">>) line 1181
2019-10-24 15:56:27.399 [error] <0.308.0>@Elixir.GenServer:whereis:1181 Supervisor {<0.308.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.310.0> exit with reason no function clause matching 'Elixir.GenServer':whereis(<<"czktd">>) line 1181 in context child_terminated
2019-10-24 15:56:27.399 [error] <0.313.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.313.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(<<"fgbcd">>) line 1181
2019-10-24 15:56:27.399 [error] <0.308.0>@Elixir.GenServer:whereis:1181 Supervisor {<0.308.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.312.0> exit with reason no function clause matching 'Elixir.GenServer':whereis(<<"nackj">>) line 1181 in context child_terminated
2019-10-24 15:56:27.399 [error] <0.308.0>@Elixir.GenServer:whereis:1181 Supervisor {<0.308.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.311.0> exit with reason no function clause matching 'Elixir.GenServer':whereis(<<"sinfl">>) line 1181 in context child_terminated
2019-10-24 15:56:27.400 [error] <0.308.0>@Elixir.GenServer:whereis:1181 Supervisor {<0.308.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.313.0> exit with reason no function clause matching 'Elixir.GenServer':whereis(<<"fgbcd">>) line 1181 in context child_terminated
2019-10-24 15:57:08.329 [error] <0.4254.0>@observer_wx:init:106 CRASH REPORT Process <0.4254.0> with 0 neighbours exited with reason: bad argument in call to erlang:register(observer, <0.4254.0>) in observer_wx:init/1 line 106
2019-10-25 14:28:08.440 [error] <0.285.0>@Elixir.KafkaEx.Protocol.ApiVersions:parse_response:62 CRASH REPORT Process <0.285.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.KafkaEx.Protocol.ApiVersions':parse_response(nil, 0) line 62
2019-10-25 14:28:08.440 [error] <0.282.0> CRASH REPORT Process <0.282.0> with 0 neighbours exited with reason: {{function_clause,[{'Elixir.KafkaEx.Protocol.ApiVersions',parse_response,[nil,0],[{file,"lib/kafka_ex/protocol/api_versions.ex"},{line,62}]},{'Elixir.KafkaEx.Server0P10AndLater',kafka_api_versions,1,[{file,"lib/kafka_ex/server_0_p_10_and_later.ex"},{line,178}]},{'Elixir.KafkaEx.Server0P10AndLater',kafka_server_init,1,[{file,"lib/kafka_ex/server_0_p_10_and_later.ex"},{line,99}]},{gen_server,init_it,2,[{file,"gen_server.erl"},{line,374}]},{gen_server,init_it,6,[{file,"gen_server.erl"},{line,342}]},...]},...} in application_master:init/4 line 138
2019-10-25 14:28:12.486 [error] <0.285.0>@Elixir.KafkaEx.Protocol.ApiVersions:parse_response:62 CRASH REPORT Process <0.285.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.KafkaEx.Protocol.ApiVersions':parse_response(nil, 0) line 62
2019-10-25 14:28:14.264 [error] <0.285.0>@Elixir.KafkaEx.Protocol.ApiVersions:parse_response:62 CRASH REPORT Process <0.285.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.KafkaEx.Protocol.ApiVersions':parse_response(nil, 0) line 62
2019-10-25 14:30:07.266 [error] <0.317.0>@Elixir.GenServer:whereis:1181 gen_server <0.317.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(<<"pjfug">>) line 1181
2019-10-25 14:30:07.266 [error] <0.309.0>@Elixir.GenServer:whereis:1181 gen_server <0.309.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(<<"avtnj">>) line 1181
2019-10-25 14:30:07.266 [error] <0.318.0>@Elixir.GenServer:whereis:1181 gen_server <0.318.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(<<"nszef">>) line 1181
2019-10-25 14:30:07.266 [error] <0.316.0>@Elixir.GenServer:whereis:1181 gen_server <0.316.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(<<"lchin">>) line 1181
2019-10-25 14:30:07.266 [error] <0.315.0>@Elixir.GenServer:whereis:1181 gen_server <0.315.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(<<"vkdzr">>) line 1181
2019-10-25 14:30:07.266 [error] <0.310.0>@Elixir.GenServer:whereis:1181 gen_server <0.310.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(<<"ihszm">>) line 1181
2019-10-25 14:30:07.266 [error] <0.314.0>@Elixir.GenServer:whereis:1181 gen_server <0.314.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(<<"lmfhn">>) line 1181
2019-10-25 14:30:07.266 [error] <0.312.0>@Elixir.GenServer:whereis:1181 gen_server <0.312.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(<<"kjfol">>) line 1181
2019-10-25 14:30:07.266 [error] <0.311.0>@Elixir.GenServer:whereis:1181 gen_server <0.311.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(<<"rbukh">>) line 1181
2019-10-25 14:30:07.266 [error] <0.313.0>@Elixir.GenServer:whereis:1181 gen_server <0.313.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(<<"zbdpg">>) line 1181
2019-10-25 14:30:07.278 [error] <0.310.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.310.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(<<"ihszm">>) line 1181
2019-10-25 14:30:07.278 [error] <0.309.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.309.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(<<"avtnj">>) line 1181
2019-10-25 14:30:07.279 [error] <0.308.0>@Elixir.GenServer:whereis:1181 Supervisor {<0.308.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.310.0> exit with reason no function clause matching 'Elixir.GenServer':whereis(<<"ihszm">>) line 1181 in context child_terminated
2019-10-25 14:30:07.279 [error] <0.308.0>@Elixir.GenServer:whereis:1181 Supervisor {<0.308.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.309.0> exit with reason no function clause matching 'Elixir.GenServer':whereis(<<"avtnj">>) line 1181 in context child_terminated
2019-10-25 14:30:07.279 [error] <0.318.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.318.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(<<"nszef">>) line 1181
2019-10-25 14:30:07.279 [error] <0.314.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.314.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(<<"lmfhn">>) line 1181
2019-10-25 14:30:07.279 [error] <0.311.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.311.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(<<"rbukh">>) line 1181
2019-10-25 14:30:07.279 [error] <0.317.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.317.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(<<"pjfug">>) line 1181
2019-10-25 14:30:07.279 [error] <0.316.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.316.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(<<"lchin">>) line 1181
2019-10-25 14:30:07.279 [error] <0.312.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.312.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(<<"kjfol">>) line 1181
2019-10-25 14:30:07.279 [error] <0.315.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.315.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(<<"vkdzr">>) line 1181
2019-10-25 14:30:07.279 [error] <0.313.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.313.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(<<"zbdpg">>) line 1181
2019-10-25 14:30:07.279 [error] <0.308.0>@Elixir.GenServer:whereis:1181 Supervisor {<0.308.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.318.0> exit with reason no function clause matching 'Elixir.GenServer':whereis(<<"nszef">>) line 1181 in context child_terminated
2019-10-25 14:30:07.279 [error] <0.308.0>@Elixir.GenServer:whereis:1181 Supervisor {<0.308.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.314.0> exit with reason no function clause matching 'Elixir.GenServer':whereis(<<"lmfhn">>) line 1181 in context child_terminated
2019-10-25 14:30:07.280 [error] <0.308.0>@Elixir.GenServer:whereis:1181 Supervisor {<0.308.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.311.0> exit with reason no function clause matching 'Elixir.GenServer':whereis(<<"rbukh">>) line 1181 in context child_terminated
2019-10-25 14:30:07.280 [error] <0.308.0>@Elixir.GenServer:whereis:1181 Supervisor {<0.308.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.317.0> exit with reason no function clause matching 'Elixir.GenServer':whereis(<<"pjfug">>) line 1181 in context child_terminated
2019-10-25 14:30:07.430 [error] <0.308.0>@Elixir.GenServer:whereis:1181 Supervisor {<0.308.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.316.0> exit with reason no function clause matching 'Elixir.GenServer':whereis(<<"lchin">>) line 1181 in context child_terminated
2019-10-25 14:30:07.430 [error] <0.308.0>@Elixir.GenServer:whereis:1181 Supervisor {<0.308.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.312.0> exit with reason no function clause matching 'Elixir.GenServer':whereis(<<"kjfol">>) line 1181 in context child_terminated
2019-10-25 14:30:07.430 [error] <0.308.0>@Elixir.GenServer:whereis:1181 Supervisor {<0.308.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.315.0> exit with reason no function clause matching 'Elixir.GenServer':whereis(<<"vkdzr">>) line 1181 in context child_terminated
2019-10-25 14:30:07.431 [error] <0.308.0>@Elixir.GenServer:whereis:1181 Supervisor {<0.308.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.313.0> exit with reason no function clause matching 'Elixir.GenServer':whereis(<<"zbdpg">>) line 1181 in context child_terminated
2019-10-25 14:30:38.147 [error] <0.428.0>@Elixir.GenServer:whereis:1181 gen_server <0.428.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(<<"kgors">>) line 1181
2019-10-25 14:30:38.149 [error] <0.436.0>@Elixir.GenServer:whereis:1181 gen_server <0.436.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(<<"bkase">>) line 1181
2019-10-25 14:30:38.149 [error] <0.429.0>@Elixir.GenServer:whereis:1181 gen_server <0.429.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(<<"rnjge">>) line 1181
2019-10-25 14:30:38.149 [error] <0.435.0>@Elixir.GenServer:whereis:1181 gen_server <0.435.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(<<"jinzg">>) line 1181
2019-10-25 14:30:38.149 [error] <0.433.0>@Elixir.GenServer:whereis:1181 gen_server <0.433.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(<<"dknzl">>) line 1181
2019-10-25 14:30:38.149 [error] <0.431.0>@Elixir.GenServer:whereis:1181 gen_server <0.431.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(<<"aintm">>) line 1181
2019-10-25 14:30:38.149 [error] <0.430.0>@Elixir.GenServer:whereis:1181 gen_server <0.430.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(<<"fczpl">>) line 1181
2019-10-25 14:30:38.149 [error] <0.428.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.428.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(<<"kgors">>) line 1181
2019-10-25 14:30:38.149 [error] <0.434.0>@Elixir.GenServer:whereis:1181 gen_server <0.434.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(<<"pkurm">>) line 1181
2019-10-25 14:30:38.150 [error] <0.427.0>@Elixir.GenServer:whereis:1181 gen_server <0.427.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(<<"cubji">>) line 1181
2019-10-25 14:30:38.150 [error] <0.432.0>@Elixir.GenServer:whereis:1181 gen_server <0.432.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(<<"pgfcv">>) line 1181
2019-10-25 14:30:38.150 [error] <0.430.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.430.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(<<"fczpl">>) line 1181
2019-10-25 14:30:38.150 [error] <0.308.0>@Elixir.GenServer:whereis:1181 Supervisor {<0.308.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.428.0> exit with reason no function clause matching 'Elixir.GenServer':whereis(<<"kgors">>) line 1181 in context child_terminated
2019-10-25 14:30:38.150 [error] <0.433.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.433.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(<<"dknzl">>) line 1181
2019-10-25 14:30:38.150 [error] <0.435.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.435.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(<<"jinzg">>) line 1181
2019-10-25 14:30:38.151 [error] <0.436.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.436.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(<<"bkase">>) line 1181
2019-10-25 14:30:38.151 [error] <0.431.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.431.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(<<"aintm">>) line 1181
2019-10-25 14:30:38.151 [error] <0.308.0>@Elixir.GenServer:whereis:1181 Supervisor {<0.308.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.430.0> exit with reason no function clause matching 'Elixir.GenServer':whereis(<<"fczpl">>) line 1181 in context child_terminated
2019-10-25 14:30:38.151 [error] <0.308.0>@Elixir.GenServer:whereis:1181 Supervisor {<0.308.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.433.0> exit with reason no function clause matching 'Elixir.GenServer':whereis(<<"dknzl">>) line 1181 in context child_terminated
2019-10-25 14:30:38.151 [error] <0.308.0>@Elixir.GenServer:whereis:1181 Supervisor {<0.308.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.435.0> exit with reason no function clause matching 'Elixir.GenServer':whereis(<<"jinzg">>) line 1181 in context child_terminated
2019-10-25 14:30:38.151 [error] <0.308.0>@Elixir.GenServer:whereis:1181 Supervisor {<0.308.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.436.0> exit with reason no function clause matching 'Elixir.GenServer':whereis(<<"bkase">>) line 1181 in context child_terminated
2019-10-25 14:30:38.151 [error] <0.308.0>@Elixir.GenServer:whereis:1181 Supervisor {<0.308.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.431.0> exit with reason no function clause matching 'Elixir.GenServer':whereis(<<"aintm">>) line 1181 in context child_terminated
2019-10-25 14:30:38.151 [error] <0.427.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.427.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(<<"cubji">>) line 1181
2019-10-25 14:30:38.151 [error] <0.308.0>@Elixir.GenServer:whereis:1181 Supervisor {<0.308.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.427.0> exit with reason no function clause matching 'Elixir.GenServer':whereis(<<"cubji">>) line 1181 in context child_terminated
2019-10-25 14:30:38.152 [error] <0.429.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.429.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(<<"rnjge">>) line 1181
2019-10-25 14:30:38.152 [error] <0.308.0>@Elixir.GenServer:whereis:1181 Supervisor {<0.308.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.429.0> exit with reason no function clause matching 'Elixir.GenServer':whereis(<<"rnjge">>) line 1181 in context child_terminated
2019-10-25 14:30:38.152 [error] <0.432.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.432.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(<<"pgfcv">>) line 1181
2019-10-25 14:30:38.152 [error] <0.308.0>@Elixir.GenServer:whereis:1181 Supervisor {<0.308.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.432.0> exit with reason no function clause matching 'Elixir.GenServer':whereis(<<"pgfcv">>) line 1181 in context child_terminated
2019-10-25 14:30:38.152 [error] <0.434.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.434.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(<<"pkurm">>) line 1181
2019-10-25 14:30:38.152 [error] <0.308.0>@Elixir.GenServer:whereis:1181 Supervisor {<0.308.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.434.0> exit with reason no function clause matching 'Elixir.GenServer':whereis(<<"pkurm">>) line 1181 in context child_terminated
2019-10-25 14:31:39.230 [error] <0.314.0>@Elixir.GenServer:whereis:1181 gen_server <0.314.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(<<"tdfoi">>) line 1181
2019-10-25 14:31:39.237 [error] <0.314.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.314.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(<<"tdfoi">>) line 1181
2019-10-25 14:31:39.237 [error] <0.313.0>@Elixir.GenServer:whereis:1181 Supervisor {<0.313.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.314.0> exit with reason no function clause matching 'Elixir.GenServer':whereis(<<"tdfoi">>) line 1181 in context child_terminated
2019-10-25 14:31:41.238 [error] <0.416.0>@Elixir.GenServer:whereis:1181 gen_server <0.416.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(<<"sinld">>) line 1181
2019-10-25 14:31:41.239 [error] <0.416.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.416.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(<<"sinld">>) line 1181
2019-10-25 14:31:41.240 [error] <0.313.0>@Elixir.GenServer:whereis:1181 Supervisor {<0.313.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.416.0> exit with reason no function clause matching 'Elixir.GenServer':whereis(<<"sinld">>) line 1181 in context child_terminated
2019-10-25 14:31:43.241 [error] <0.417.0>@Elixir.GenServer:whereis:1181 gen_server <0.417.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(<<"pmdvg">>) line 1181
2019-10-25 14:31:43.242 [error] <0.417.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.417.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(<<"pmdvg">>) line 1181
2019-10-25 14:31:43.242 [error] <0.313.0>@Elixir.GenServer:whereis:1181 Supervisor {<0.313.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.417.0> exit with reason no function clause matching 'Elixir.GenServer':whereis(<<"pmdvg">>) line 1181 in context child_terminated
2019-10-25 14:37:12.847 [error] <0.307.0>@Elixir.GenServer:whereis:1181 gen_server <0.307.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(<<"ueokm">>) line 1181
2019-10-25 14:37:12.847 [error] <0.309.0>@Elixir.GenServer:whereis:1181 gen_server <0.309.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(<<"lomhr">>) line 1181
2019-10-25 14:37:12.847 [error] <0.2969.0>@Elixir.GenServer:whereis:1181 gen_server <0.2969.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(<<"bcgtz">>) line 1181
2019-10-25 14:37:12.847 [error] <0.310.0>@Elixir.GenServer:whereis:1181 gen_server <0.310.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(<<"nsbjp">>) line 1181
2019-10-25 14:37:12.847 [error] <0.308.0>@Elixir.GenServer:whereis:1181 gen_server <0.308.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(<<"nrfki">>) line 1181
2019-10-25 14:37:12.847 [error] <0.2965.0>@Elixir.GenServer:whereis:1181 gen_server <0.2965.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(<<"onfit">>) line 1181
2019-10-25 14:37:12.847 [error] <0.2968.0>@Elixir.GenServer:whereis:1181 gen_server <0.2968.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(<<"kscgp">>) line 1181
2019-10-25 14:37:12.847 [error] <0.2967.0>@Elixir.GenServer:whereis:1181 gen_server <0.2967.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(<<"snpek">>) line 1181
2019-10-25 14:37:12.847 [error] <0.306.0>@Elixir.GenServer:whereis:1181 gen_server <0.306.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(<<"dlgcm">>) line 1181
2019-10-25 14:37:12.860 [error] <0.309.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.309.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(<<"lomhr">>) line 1181
2019-10-25 14:37:12.860 [error] <0.2969.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.2969.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(<<"bcgtz">>) line 1181
2019-10-25 14:37:12.860 [error] <0.310.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.310.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(<<"nsbjp">>) line 1181
2019-10-25 14:37:12.860 [error] <0.305.0>@Elixir.GenServer:whereis:1181 Supervisor {<0.305.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.309.0> exit with reason no function clause matching 'Elixir.GenServer':whereis(<<"lomhr">>) line 1181 in context child_terminated
2019-10-25 14:37:12.861 [error] <0.307.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.307.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(<<"ueokm">>) line 1181
2019-10-25 14:37:12.861 [error] <0.308.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.308.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(<<"nrfki">>) line 1181
2019-10-25 14:37:12.861 [error] <0.2965.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.2965.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(<<"onfit">>) line 1181
2019-10-25 14:37:12.861 [error] <0.306.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.306.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(<<"dlgcm">>) line 1181
2019-10-25 14:37:12.861 [error] <0.2968.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.2968.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(<<"kscgp">>) line 1181
2019-10-25 14:37:12.861 [error] <0.305.0>@Elixir.GenServer:whereis:1181 Supervisor {<0.305.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.2969.0> exit with reason no function clause matching 'Elixir.GenServer':whereis(<<"bcgtz">>) line 1181 in context child_terminated
2019-10-25 14:37:12.861 [error] <0.2967.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.2967.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(<<"snpek">>) line 1181
2019-10-25 14:37:12.861 [error] <0.305.0>@Elixir.GenServer:whereis:1181 Supervisor {<0.305.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.310.0> exit with reason no function clause matching 'Elixir.GenServer':whereis(<<"nsbjp">>) line 1181 in context child_terminated
2019-10-25 14:37:12.861 [error] <0.305.0>@Elixir.GenServer:whereis:1181 Supervisor {<0.305.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.307.0> exit with reason no function clause matching 'Elixir.GenServer':whereis(<<"ueokm">>) line 1181 in context child_terminated
2019-10-25 14:37:12.861 [error] <0.305.0>@Elixir.GenServer:whereis:1181 Supervisor {<0.305.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.308.0> exit with reason no function clause matching 'Elixir.GenServer':whereis(<<"nrfki">>) line 1181 in context child_terminated
2019-10-25 14:37:12.861 [error] <0.305.0>@Elixir.GenServer:whereis:1181 Supervisor {<0.305.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.2965.0> exit with reason no function clause matching 'Elixir.GenServer':whereis(<<"onfit">>) line 1181 in context child_terminated
2019-10-25 14:37:12.862 [error] <0.305.0>@Elixir.GenServer:whereis:1181 Supervisor {<0.305.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.306.0> exit with reason no function clause matching 'Elixir.GenServer':whereis(<<"dlgcm">>) line 1181 in context child_terminated
2019-10-25 14:37:12.862 [error] <0.305.0>@Elixir.GenServer:whereis:1181 Supervisor {<0.305.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.2968.0> exit with reason no function clause matching 'Elixir.GenServer':whereis(<<"kscgp">>) line 1181 in context child_terminated
2019-10-25 14:37:12.862 [error] <0.305.0>@Elixir.GenServer:whereis:1181 Supervisor {<0.305.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.2967.0> exit with reason no function clause matching 'Elixir.GenServer':whereis(<<"snpek">>) line 1181 in context child_terminated
2019-10-25 14:37:15.654 [error] <0.2970.0>@Elixir.GenServer:whereis:1181 gen_server <0.2970.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(<<"nheoc">>) line 1181
2019-10-25 14:37:15.655 [error] <0.2973.0>@Elixir.GenServer:whereis:1181 gen_server <0.2973.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(<<"czidr">>) line 1181
2019-10-25 14:37:15.655 [error] <0.2972.0>@Elixir.GenServer:whereis:1181 gen_server <0.2972.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(<<"ublmg">>) line 1181
2019-10-25 14:37:15.655 [error] <0.2974.0>@Elixir.GenServer:whereis:1181 gen_server <0.2974.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(<<"uirtg">>) line 1181
2019-10-25 14:37:15.656 [error] <0.2973.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.2973.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(<<"czidr">>) line 1181
2019-10-25 14:37:15.656 [error] <0.2971.0>@Elixir.GenServer:whereis:1181 gen_server <0.2971.0> terminated with reason: no function clause matching 'Elixir.GenServer':whereis(<<"lcapd">>) line 1181
2019-10-25 14:37:15.656 [error] <0.305.0>@Elixir.GenServer:whereis:1181 Supervisor {<0.305.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.2973.0> exit with reason no function clause matching 'Elixir.GenServer':whereis(<<"czidr">>) line 1181 in context child_terminated
2019-10-25 14:37:15.657 [error] <0.2972.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.2972.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(<<"ublmg">>) line 1181
2019-10-25 14:37:15.657 [error] <0.2970.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.2970.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(<<"nheoc">>) line 1181
2019-10-25 14:37:15.657 [error] <0.305.0>@Elixir.GenServer:whereis:1181 Supervisor {<0.305.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.2972.0> exit with reason no function clause matching 'Elixir.GenServer':whereis(<<"ublmg">>) line 1181 in context child_terminated
2019-10-25 14:37:15.657 [error] <0.305.0>@Elixir.GenServer:whereis:1181 Supervisor {<0.305.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.2970.0> exit with reason no function clause matching 'Elixir.GenServer':whereis(<<"nheoc">>) line 1181 in context child_terminated
2019-10-25 14:37:15.657 [error] <0.2974.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.2974.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(<<"uirtg">>) line 1181
2019-10-25 14:37:15.657 [error] <0.2971.0>@Elixir.GenServer:whereis:1181 CRASH REPORT Process <0.2971.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.GenServer':whereis(<<"lcapd">>) line 1181
2019-10-25 14:37:15.657 [error] <0.305.0>@Elixir.GenServer:whereis:1181 Supervisor {<0.305.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.2974.0> exit with reason no function clause matching 'Elixir.GenServer':whereis(<<"uirtg">>) line 1181 in context child_terminated
2019-10-25 14:37:15.658 [error] <0.994.0>@appmon_info:do_work:308 gen_server appmon_info terminated with reason: no match of right hand value {error,{'EXIT',{badarg,[{erlang,element,[2,undefined],[]},{appmon_info,check_sasl_ancestor,2,[{file,"appmon_info.erl"},{line,618}]},{appmon_info,maybe_add_child_sasl,4,[{file,"appmon_info.erl"},{line,590}]},{lists,foldr,3,[{file,"lists.erl"},{line,1276}]},{appmon_info,do_find_proc2,5,[{file,"appmon_info.erl"},{line,497}]},{appmon_info,calc_app_tree,2,[{file,"appmon_info.erl"},{line,450}]},{appmon_info,do_work,2,[{file,"appmon_info.erl"},{line,308}]},{appmon_info,handle_info,2,[{file,"appmon..."},...]},...]}}} in appmon_info:do_work/2 line 308
2019-10-25 14:37:15.658 [error] <0.305.0>@Elixir.GenServer:whereis:1181 Supervisor {<0.305.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.2971.0> exit with reason no function clause matching 'Elixir.GenServer':whereis(<<"lcapd">>) line 1181 in context child_terminated
2019-10-25 14:37:15.673 [error] <0.994.0>@appmon_info:do_work:308 CRASH REPORT Process appmon_info with 0 neighbours crashed with reason: no match of right hand value {error,{'EXIT',{badarg,[{erlang,element,[2,undefined],[]},{appmon_info,check_sasl_ancestor,2,[{file,"appmon_info.erl"},{line,618}]},{appmon_info,maybe_add_child_sasl,4,[{file,"appmon_info.erl"},{line,590}]},{lists,foldr,3,[{file,"lists.erl"},{line,1276}]},{appmon_info,do_find_proc2,5,[{file,"appmon_info.erl"},{line,497}]},{appmon_info,calc_app_tree,2,[{file,"appmon_info.erl"},{line,450}]},{appmon_info,do_work,2,[{file,"appmon_info.erl"},{line,308}]},{appmon_info,handle_info,2,[{file,"appmon..."},...]},...]}}} in appmon_info:do_work/2 line 308
2019-10-25 14:48:07.727 [error] <0.318.0>@Elixir.KafkaEx.Protocol.Produce:create_request:71 gen_server ejvoc terminated with reason: bad argument in call to erlang:byte_size([<<"bmw-telematics">>]) in 'Elixir.KafkaEx.Protocol.Produce':create_request/3 line 71
2019-10-25 14:48:07.727 [error] <0.312.0>@Elixir.KafkaEx.Protocol.Produce:create_request:71 gen_server fnvma terminated with reason: bad argument in call to erlang:byte_size([<<"bmw-telematics">>]) in 'Elixir.KafkaEx.Protocol.Produce':create_request/3 line 71
2019-10-25 14:48:07.727 [error] <0.310.0>@Elixir.KafkaEx.Protocol.Produce:create_request:71 gen_server uhzkd terminated with reason: bad argument in call to erlang:byte_size([<<"bmw-telematics">>]) in 'Elixir.KafkaEx.Protocol.Produce':create_request/3 line 71
2019-10-25 14:48:07.728 [error] <0.314.0>@Elixir.KafkaEx.Protocol.Produce:create_request:71 gen_server tmegh terminated with reason: bad argument in call to erlang:byte_size([<<"bmw-telematics">>]) in 'Elixir.KafkaEx.Protocol.Produce':create_request/3 line 71
2019-10-25 14:48:07.728 [error] <0.316.0>@Elixir.KafkaEx.Protocol.Produce:create_request:71 gen_server crpjs terminated with reason: bad argument in call to erlang:byte_size([<<"bmw-telematics">>]) in 'Elixir.KafkaEx.Protocol.Produce':create_request/3 line 71
2019-10-25 14:48:07.728 [error] <0.8466.0>@Elixir.KafkaEx.Protocol.Produce:create_request:71 gen_server htdka terminated with reason: bad argument in call to erlang:byte_size([<<"bmw-telematics">>]) in 'Elixir.KafkaEx.Protocol.Produce':create_request/3 line 71
2019-10-25 14:48:07.736 [error] <0.8469.0>@Elixir.KafkaEx.Protocol.Produce:create_request:71 gen_server fzkre terminated with reason: bad argument in call to erlang:byte_size([<<"bmw-telematics">>]) in 'Elixir.KafkaEx.Protocol.Produce':create_request/3 line 71
2019-10-25 14:48:07.744 [error] <0.8471.0>@Elixir.KafkaEx.Protocol.Produce:create_request:71 gen_server hjdnf terminated with reason: bad argument in call to erlang:byte_size([<<"bmw-telematics">>]) in 'Elixir.KafkaEx.Protocol.Produce':create_request/3 line 71
2019-10-25 14:48:07.753 [error] <0.8473.0>@Elixir.KafkaEx.Protocol.Produce:create_request:71 gen_server nusmk terminated with reason: bad argument in call to erlang:byte_size([<<"bmw-telematics">>]) in 'Elixir.KafkaEx.Protocol.Produce':create_request/3 line 71
2019-10-25 14:48:07.768 [error] <0.310.0>@Elixir.KafkaEx.Protocol.Produce:create_request:71 CRASH REPORT Process uhzkd with 0 neighbours crashed with reason: bad argument in call to erlang:byte_size([<<"bmw-telematics">>]) in 'Elixir.KafkaEx.Protocol.Produce':create_request/3 line 71
2019-10-25 14:48:07.769 [error] <0.296.0>@Elixir.KafkaEx.Protocol.Produce:create_request:71 Supervisor 'Elixir.KafkaEx.Supervisor' had child 'Elixir.KafkaEx.Server0P10AndLater' started with 'Elixir.KafkaEx.Server0P10AndLater':start_link([{uris,[{<<"localhost">>,9092}]},{consumer_group,<<"dev-bmw">>},{use_ssl,false},{ssl_options,[]}], uhzkd) at <0.310.0> exit with reason bad argument in call to erlang:byte_size([<<"bmw-telematics">>]) in 'Elixir.KafkaEx.Protocol.Produce':create_request/3 line 71 in context child_terminated
2019-10-25 14:48:07.769 [error] <0.8473.0>@Elixir.KafkaEx.Protocol.Produce:create_request:71 CRASH REPORT Process nusmk with 0 neighbours crashed with reason: bad argument in call to erlang:byte_size([<<"bmw-telematics">>]) in 'Elixir.KafkaEx.Protocol.Produce':create_request/3 line 71
2019-10-25 14:48:07.769 [error] <0.8471.0>@Elixir.KafkaEx.Protocol.Produce:create_request:71 CRASH REPORT Process hjdnf with 0 neighbours crashed with reason: bad argument in call to erlang:byte_size([<<"bmw-telematics">>]) in 'Elixir.KafkaEx.Protocol.Produce':create_request/3 line 71
2019-10-25 14:48:07.769 [error] <0.312.0>@Elixir.KafkaEx.Protocol.Produce:create_request:71 CRASH REPORT Process fnvma with 0 neighbours crashed with reason: bad argument in call to erlang:byte_size([<<"bmw-telematics">>]) in 'Elixir.KafkaEx.Protocol.Produce':create_request/3 line 71
2019-10-25 14:48:07.769 [error] <0.309.0>@Elixir.KafkaEx.Protocol.Produce:create_request:71 gen_server <0.309.0> terminated with reason: bad argument in call to erlang:byte_size([<<"bmw-telematics">>]) in 'Elixir.KafkaEx.Protocol.Produce':create_request/3 line 71 in 'Elixir.GenServer':call/3 line 1009
2019-10-25 14:48:07.770 [error] <0.318.0>@Elixir.KafkaEx.Protocol.Produce:create_request:71 CRASH REPORT Process ejvoc with 0 neighbours crashed with reason: bad argument in call to erlang:byte_size([<<"bmw-telematics">>]) in 'Elixir.KafkaEx.Protocol.Produce':create_request/3 line 71
2019-10-25 14:48:07.770 [error] <0.314.0>@Elixir.KafkaEx.Protocol.Produce:create_request:71 CRASH REPORT Process tmegh with 0 neighbours crashed with reason: bad argument in call to erlang:byte_size([<<"bmw-telematics">>]) in 'Elixir.KafkaEx.Protocol.Produce':create_request/3 line 71
2019-10-25 14:48:07.770 [error] <0.311.0>@Elixir.KafkaEx.Protocol.Produce:create_request:71 gen_server <0.311.0> terminated with reason: bad argument in call to erlang:byte_size([<<"bmw-telematics">>]) in 'Elixir.KafkaEx.Protocol.Produce':create_request/3 line 71 in 'Elixir.GenServer':call/3 line 1009
2019-10-25 14:48:07.770 [error] <0.8466.0>@Elixir.KafkaEx.Protocol.Produce:create_request:71 CRASH REPORT Process htdka with 0 neighbours crashed with reason: bad argument in call to erlang:byte_size([<<"bmw-telematics">>]) in 'Elixir.KafkaEx.Protocol.Produce':create_request/3 line 71
2019-10-25 14:48:07.770 [error] <0.8470.0>@Elixir.KafkaEx.Protocol.Produce:create_request:71 gen_server <0.8470.0> terminated with reason: bad argument in call to erlang:byte_size([<<"bmw-telematics">>]) in 'Elixir.KafkaEx.Protocol.Produce':create_request/3 line 71 in 'Elixir.GenServer':call/3 line 1009
2019-10-25 14:48:07.771 [error] <0.8472.0>@Elixir.KafkaEx.Protocol.Produce:create_request:71 gen_server <0.8472.0> terminated with reason: bad argument in call to erlang:byte_size([<<"bmw-telematics">>]) in 'Elixir.KafkaEx.Protocol.Produce':create_request/3 line 71 in 'Elixir.GenServer':call/3 line 1009
2019-10-25 14:48:07.771 [error] <0.8465.0>@Elixir.KafkaEx.Protocol.Produce:create_request:71 gen_server <0.8465.0> terminated with reason: bad argument in call to erlang:byte_size([<<"bmw-telematics">>]) in 'Elixir.KafkaEx.Protocol.Produce':create_request/3 line 71 in 'Elixir.GenServer':call/3 line 1009
2019-10-25 14:48:07.771 [error] <0.313.0>@Elixir.KafkaEx.Protocol.Produce:create_request:71 gen_server <0.313.0> terminated with reason: bad argument in call to erlang:byte_size([<<"bmw-telematics">>]) in 'Elixir.KafkaEx.Protocol.Produce':create_request/3 line 71 in 'Elixir.GenServer':call/3 line 1009
2019-10-25 14:48:07.771 [error] <0.309.0>@Elixir.KafkaEx.Protocol.Produce:create_request:71 CRASH REPORT Process <0.309.0> with 0 neighbours exited with reason: bad argument in call to erlang:byte_size([<<"bmw-telematics">>]) in 'Elixir.KafkaEx.Protocol.Produce':create_request/3 line 71 in 'Elixir.GenServer':call/3 line 1009
2019-10-25 14:48:07.771 [error] <0.8469.0>@Elixir.KafkaEx.Protocol.Produce:create_request:71 CRASH REPORT Process fzkre with 0 neighbours crashed with reason: bad argument in call to erlang:byte_size([<<"bmw-telematics">>]) in 'Elixir.KafkaEx.Protocol.Produce':create_request/3 line 71
2019-10-25 14:48:07.771 [error] <0.308.0>@Elixir.KafkaEx.Protocol.Produce:create_request:71 Supervisor {<0.308.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.309.0> exit with reason bad argument in call to erlang:byte_size([<<"bmw-telematics">>]) in 'Elixir.KafkaEx.Protocol.Produce':create_request/3 line 71 in context child_terminated
2019-10-25 14:48:07.771 [error] <0.8470.0>@Elixir.KafkaEx.Protocol.Produce:create_request:71 CRASH REPORT Process <0.8470.0> with 0 neighbours exited with reason: bad argument in call to erlang:byte_size([<<"bmw-telematics">>]) in 'Elixir.KafkaEx.Protocol.Produce':create_request/3 line 71 in 'Elixir.GenServer':call/3 line 1009
2019-10-25 14:48:07.771 [error] <0.311.0>@Elixir.KafkaEx.Protocol.Produce:create_request:71 CRASH REPORT Process <0.311.0> with 0 neighbours exited with reason: bad argument in call to erlang:byte_size([<<"bmw-telematics">>]) in 'Elixir.KafkaEx.Protocol.Produce':create_request/3 line 71 in 'Elixir.GenServer':call/3 line 1009
2019-10-25 14:48:07.772 [error] <0.8472.0>@Elixir.KafkaEx.Protocol.Produce:create_request:71 CRASH REPORT Process <0.8472.0> with 0 neighbours exited with reason: bad argument in call to erlang:byte_size([<<"bmw-telematics">>]) in 'Elixir.KafkaEx.Protocol.Produce':create_request/3 line 71 in 'Elixir.GenServer':call/3 line 1009
2019-10-25 14:48:07.772 [error] <0.8468.0>@Elixir.KafkaEx.Protocol.Produce:create_request:71 gen_server <0.8468.0> terminated with reason: bad argument in call to erlang:byte_size([<<"bmw-telematics">>]) in 'Elixir.KafkaEx.Protocol.Produce':create_request/3 line 71 in 'Elixir.GenServer':call/3 line 1009
2019-10-25 14:48:07.772 [error] <0.317.0>@Elixir.KafkaEx.Protocol.Produce:create_request:71 gen_server <0.317.0> terminated with reason: bad argument in call to erlang:byte_size([<<"bmw-telematics">>]) in 'Elixir.KafkaEx.Protocol.Produce':create_request/3 line 71 in 'Elixir.GenServer':call/3 line 1009
2019-10-25 14:48:07.772 [error] <0.8465.0>@Elixir.KafkaEx.Protocol.Produce:create_request:71 CRASH REPORT Process <0.8465.0> with 0 neighbours exited with reason: bad argument in call to erlang:byte_size([<<"bmw-telematics">>]) in 'Elixir.KafkaEx.Protocol.Produce':create_request/3 line 71 in 'Elixir.GenServer':call/3 line 1009
2019-10-25 14:48:07.772 [error] <0.316.0>@Elixir.KafkaEx.Protocol.Produce:create_request:71 CRASH REPORT Process crpjs with 0 neighbours crashed with reason: bad argument in call to erlang:byte_size([<<"bmw-telematics">>]) in 'Elixir.KafkaEx.Protocol.Produce':create_request/3 line 71
2019-10-25 14:48:07.772 [error] <0.315.0>@Elixir.KafkaEx.Protocol.Produce:create_request:71 gen_server <0.315.0> terminated with reason: bad argument in call to erlang:byte_size([<<"bmw-telematics">>]) in 'Elixir.KafkaEx.Protocol.Produce':create_request/3 line 71 in 'Elixir.GenServer':call/3 line 1009
2019-10-25 14:48:07.772 [error] <0.313.0>@Elixir.KafkaEx.Protocol.Produce:create_request:71 CRASH REPORT Process <0.313.0> with 0 neighbours exited with reason: bad argument in call to erlang:byte_size([<<"bmw-telematics">>]) in 'Elixir.KafkaEx.Protocol.Produce':create_request/3 line 71 in 'Elixir.GenServer':call/3 line 1009
2019-10-25 14:48:07.772 [error] <0.8468.0>@Elixir.KafkaEx.Protocol.Produce:create_request:71 CRASH REPORT Process <0.8468.0> with 0 neighbours exited with reason: bad argument in call to erlang:byte_size([<<"bmw-telematics">>]) in 'Elixir.KafkaEx.Protocol.Produce':create_request/3 line 71 in 'Elixir.GenServer':call/3 line 1009
2019-10-25 14:48:07.772 [error] <0.317.0>@Elixir.KafkaEx.Protocol.Produce:create_request:71 CRASH REPORT Process <0.317.0> with 0 neighbours exited with reason: bad argument in call to erlang:byte_size([<<"bmw-telematics">>]) in 'Elixir.KafkaEx.Protocol.Produce':create_request/3 line 71 in 'Elixir.GenServer':call/3 line 1009
2019-10-25 14:48:07.773 [error] <0.315.0>@Elixir.KafkaEx.Protocol.Produce:create_request:71 CRASH REPORT Process <0.315.0> with 0 neighbours exited with reason: bad argument in call to erlang:byte_size([<<"bmw-telematics">>]) in 'Elixir.KafkaEx.Protocol.Produce':create_request/3 line 71 in 'Elixir.GenServer':call/3 line 1009
2019-10-25 14:48:07.781 [error] <0.296.0>@Elixir.KafkaEx.Protocol.Produce:create_request:71 Supervisor 'Elixir.KafkaEx.Supervisor' had child 'Elixir.KafkaEx.Server0P10AndLater' started with 'Elixir.KafkaEx.Server0P10AndLater':start_link([{uris,[{<<"localhost">>,9092}]},{consumer_group,<<"dev-bmw">>},{use_ssl,false},{ssl_options,[]}], nusmk) at <0.8473.0> exit with reason bad argument in call to erlang:byte_size([<<"bmw-telematics">>]) in 'Elixir.KafkaEx.Protocol.Produce':create_request/3 line 71 in context child_terminated
2019-10-25 14:48:07.791 [error] <0.296.0>@Elixir.KafkaEx.Protocol.Produce:create_request:71 Supervisor 'Elixir.KafkaEx.Supervisor' had child 'Elixir.KafkaEx.Server0P10AndLater' started with 'Elixir.KafkaEx.Server0P10AndLater':start_link([{uris,[{<<"localhost">>,9092}]},{consumer_group,<<"dev-bmw">>},{use_ssl,false},{ssl_options,[]}], hjdnf) at <0.8471.0> exit with reason bad argument in call to erlang:byte_size([<<"bmw-telematics">>]) in 'Elixir.KafkaEx.Protocol.Produce':create_request/3 line 71 in context child_terminated
2019-10-25 14:48:07.973 [error] <0.296.0>@Elixir.KafkaEx.Protocol.Produce:create_request:71 Supervisor 'Elixir.KafkaEx.Supervisor' had child 'Elixir.KafkaEx.Server0P10AndLater' started with 'Elixir.KafkaEx.Server0P10AndLater':start_link([{uris,[{<<"localhost">>,9092}]},{consumer_group,<<"dev-bmw">>},{use_ssl,false},{ssl_options,[]}], fnvma) at <0.312.0> exit with reason bad argument in call to erlang:byte_size([<<"bmw-telematics">>]) in 'Elixir.KafkaEx.Protocol.Produce':create_request/3 line 71 in context child_terminated
2019-10-25 14:48:07.974 [error] <0.296.0>@Elixir.KafkaEx.Protocol.Produce:create_request:71 Supervisor 'Elixir.KafkaEx.Supervisor' had child 'Elixir.KafkaEx.Server0P10AndLater' started with 'Elixir.KafkaEx.Server0P10AndLater':start_link([{uris,[{<<"localhost">>,9092}]},{consumer_group,<<"dev-bmw">>},{use_ssl,false},{ssl_options,[]}], ejvoc) at <0.318.0> exit with reason bad argument in call to erlang:byte_size([<<"bmw-telematics">>]) in 'Elixir.KafkaEx.Protocol.Produce':create_request/3 line 71 in context child_terminated
2019-10-25 14:48:07.974 [error] <0.296.0>@Elixir.KafkaEx.Protocol.Produce:create_request:71 Supervisor 'Elixir.KafkaEx.Supervisor' had child 'Elixir.KafkaEx.Server0P10AndLater' started with 'Elixir.KafkaEx.Server0P10AndLater':start_link([{uris,[{<<"localhost">>,9092}]},{consumer_group,<<"dev-bmw">>},{use_ssl,false},{ssl_options,[]}], tmegh) at <0.314.0> exit with reason bad argument in call to erlang:byte_size([<<"bmw-telematics">>]) in 'Elixir.KafkaEx.Protocol.Produce':create_request/3 line 71 in context child_terminated
2019-10-25 14:48:07.974 [error] <0.296.0>@Elixir.KafkaEx.Protocol.Produce:create_request:71 Supervisor 'Elixir.KafkaEx.Supervisor' had child 'Elixir.KafkaEx.Server0P10AndLater' started with 'Elixir.KafkaEx.Server0P10AndLater':start_link([{uris,[{<<"localhost">>,9092}]},{consumer_group,<<"dev-bmw">>},{use_ssl,false},{ssl_options,[]}], htdka) at <0.8466.0> exit with reason bad argument in call to erlang:byte_size([<<"bmw-telematics">>]) in 'Elixir.KafkaEx.Protocol.Produce':create_request/3 line 71 in context child_terminated
2019-10-25 14:48:07.974 [error] <0.296.0>@Elixir.KafkaEx.Protocol.Produce:create_request:71 Supervisor 'Elixir.KafkaEx.Supervisor' had child 'Elixir.KafkaEx.Server0P10AndLater' started with 'Elixir.KafkaEx.Server0P10AndLater':start_link([{uris,[{<<"localhost">>,9092}]},{consumer_group,<<"dev-bmw">>},{use_ssl,false},{ssl_options,[]}], fzkre) at <0.8469.0> exit with reason bad argument in call to erlang:byte_size([<<"bmw-telematics">>]) in 'Elixir.KafkaEx.Protocol.Produce':create_request/3 line 71 in context child_terminated
2019-10-25 14:48:07.974 [error] <0.296.0>@Elixir.KafkaEx.Protocol.Produce:create_request:71 Supervisor 'Elixir.KafkaEx.Supervisor' had child 'Elixir.KafkaEx.Server0P10AndLater' started with 'Elixir.KafkaEx.Server0P10AndLater':start_link([{uris,[{<<"localhost">>,9092}]},{consumer_group,<<"dev-bmw">>},{use_ssl,false},{ssl_options,[]}], crpjs) at <0.316.0> exit with reason bad argument in call to erlang:byte_size([<<"bmw-telematics">>]) in 'Elixir.KafkaEx.Protocol.Produce':create_request/3 line 71 in context child_terminated
2019-10-25 14:48:07.974 [error] <0.308.0>@Elixir.KafkaEx.Protocol.Produce:create_request:71 Supervisor {<0.308.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.8470.0> exit with reason bad argument in call to erlang:byte_size([<<"bmw-telematics">>]) in 'Elixir.KafkaEx.Protocol.Produce':create_request/3 line 71 in context child_terminated
2019-10-25 14:48:07.974 [error] <0.308.0>@Elixir.KafkaEx.Protocol.Produce:create_request:71 Supervisor {<0.308.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.311.0> exit with reason bad argument in call to erlang:byte_size([<<"bmw-telematics">>]) in 'Elixir.KafkaEx.Protocol.Produce':create_request/3 line 71 in context child_terminated
2019-10-25 14:48:07.975 [error] <0.308.0>@Elixir.KafkaEx.Protocol.Produce:create_request:71 Supervisor {<0.308.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.8472.0> exit with reason bad argument in call to erlang:byte_size([<<"bmw-telematics">>]) in 'Elixir.KafkaEx.Protocol.Produce':create_request/3 line 71 in context child_terminated
2019-10-25 14:48:07.975 [error] <0.308.0>@Elixir.KafkaEx.Protocol.Produce:create_request:71 Supervisor {<0.308.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.8465.0> exit with reason bad argument in call to erlang:byte_size([<<"bmw-telematics">>]) in 'Elixir.KafkaEx.Protocol.Produce':create_request/3 line 71 in context child_terminated
2019-10-25 14:48:09.864 [error] <0.8483.0>@Elixir.KafkaEx.Protocol.Produce:create_request:71 gen_server hbgla terminated with reason: bad argument in call to erlang:byte_size([<<"bmw-telematics">>]) in 'Elixir.KafkaEx.Protocol.Produce':create_request/3 line 71
2019-10-25 14:48:09.868 [error] <0.8483.0>@Elixir.KafkaEx.Protocol.Produce:create_request:71 CRASH REPORT Process hbgla with 0 neighbours crashed with reason: bad argument in call to erlang:byte_size([<<"bmw-telematics">>]) in 'Elixir.KafkaEx.Protocol.Produce':create_request/3 line 71
2019-10-25 14:48:09.869 [error] <0.296.0>@Elixir.KafkaEx.Protocol.Produce:create_request:71 Supervisor 'Elixir.KafkaEx.Supervisor' had child 'Elixir.KafkaEx.Server0P10AndLater' started with 'Elixir.KafkaEx.Server0P10AndLater':start_link([{uris,[{<<"localhost">>,9092}]},{consumer_group,<<"dev-bmw">>},{use_ssl,false},{ssl_options,[]}], hbgla) at <0.8483.0> exit with reason bad argument in call to erlang:byte_size([<<"bmw-telematics">>]) in 'Elixir.KafkaEx.Protocol.Produce':create_request/3 line 71 in context child_terminated
2019-10-25 14:48:09.869 [error] <0.8475.0>@Elixir.KafkaEx.Protocol.Produce:create_request:71 gen_server <0.8475.0> terminated with reason: bad argument in call to erlang:byte_size([<<"bmw-telematics">>]) in 'Elixir.KafkaEx.Protocol.Produce':create_request/3 line 71 in 'Elixir.GenServer':call/3 line 1009
2019-10-25 14:48:09.871 [error] <0.8475.0>@Elixir.KafkaEx.Protocol.Produce:create_request:71 CRASH REPORT Process <0.8475.0> with 0 neighbours exited with reason: bad argument in call to erlang:byte_size([<<"bmw-telematics">>]) in 'Elixir.KafkaEx.Protocol.Produce':create_request/3 line 71 in 'Elixir.GenServer':call/3 line 1009
2019-10-25 14:48:09.871 [error] <0.308.0>@Elixir.KafkaEx.Protocol.Produce:create_request:71 Supervisor {<0.308.0>,poolboy_sup} had child 'Elixir.Kafka.Producer.Producer' started with {'Elixir.Kafka.Producer.Producer',start_link,undefined} at <0.8475.0> exit with reason bad argument in call to erlang:byte_size([<<"bmw-telematics">>]) in 'Elixir.KafkaEx.Protocol.Produce':create_request/3 line 71 in context child_terminated
2019-10-28 09:24:23.934 [error] <0.302.0>@Elixir.KafkaEx.Protocol.ApiVersions:parse_response:62 CRASH REPORT Process <0.302.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.KafkaEx.Protocol.ApiVersions':parse_response(nil, 0) line 62
2019-10-28 09:26:35.905 [error] <0.309.0>@Elixir.KafkaEx.Protocol.ApiVersions:parse_response:62 CRASH REPORT Process <0.309.0> with 0 neighbours crashed with reason: no function clause matching 'Elixir.KafkaEx.Protocol.ApiVersions':parse_response(nil, 0) line 62
